docs/source/benchmarking.md

```diff
@@ -0,0 +1,69 @@
+# Benchmarking Performance
+
+Benchmarking packet parsing is challenging because performance is greatly impacted by the complexity of the packet 
+structures being parsed. There are a few measures by which we can assess the performance of Space Packet Parser.
+
+*Note: Throughout the Space Packet Parser repo and documentation space, 
+B/kB means bytes/kilobytes and b/kb means bits/kilobits.*
```

Comment:
Format it with the Note style?
https://github.com/orgs/community/discussions/16925